infection.json.dist

```diff
@@ -0,0 +1,19 @@
+{

```

Comment:
This doesn't run in CI, does it?

Reply:
I want to create a CI. This is the next step.